### PR TITLE
[EasyCore] Including FixPostgreSqlDefaultSchemaListener to package

### PR DESCRIPTION
**Pull request by DKeeper**

### CODE REVIEW

**Comment by roman-eonx** [comment]
@DKeeper please add a `docs/doctrine/postgresql_migrations_fix.md` file with a short description of what issue does this listener fix and how to apply it.

File: packages/EasyCore/src/Doctrine/Events/FixPostgreSqlDefaultSchemaListener.php [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+namespace EonX\EasyCore\Doctrine\Events;
+
+use Doctrine\DBAL\Schema\PostgreSqlSchemaManager;
+use Doctrine\ORM\Tools\Event\GenerateSchemaEventArgs;
+
+/**
+ * Workaround for an issue: https://github.com/doctrine/dbal/issues/1110.
+ *
+ * @codeCoverageIgnore
```
Comment (roman-eonx) [inline packages/EasyCore/src/Doctrine/Events/FixPostgreSqlDefaultSchemaListener.php:13]: Please add an `@author https://github.com/vudaltsov` annotation to mention the original author from the issue above.

**Review by roman-eonx — CHANGES_REQUESTED** [review]

**Comment by DKeeper** [comment]
> @DKeeper please add a `docs/doctrine/postgresql_migrations_fix.md` file with a short description of what issue does this listener fix and how to apply it.

Done.

File: packages/EasyCore/src/Doctrine/Events/FixPostgreSqlDefaultSchemaListener.php (hunk 1 above)
Comment (DKeeper) [reply packages/EasyCore/src/Doctrine/Events/FixPostgreSqlDefaultSchemaListener.php:15]: Done.

**Review by roman-eonx — APPROVED** [review]

**Review by BOB41K1987 — APPROVED** [review]

**Review by natepage — APPROVED** [review]

**Pull request merged**